### ISSUE TITLE
"Tag" doesn't group messages

### ISSUE THREAD

**Issue by gmoraiz** [phonegap/phonegap-plugin-push#2523]
### Expected Behaviour
 $data = array(
            "title"       => $me->name,
            "message"     => "📰 " . $publication->description, //📷🔗📽️
            "tag"         => "new_messages",
            "notId"       => $me->id,
            "priority"    => 2,
            "icon"        => "notification_small_icon",
            "color"       => "#867446",
            "summaryText" => "Você tem %n% publicações novas",
            "style"       => "inbox",
            "userId"      => $me->id,
);
#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
Android 7.0.0
#### (Android) What device vendor (e.g. Samsung, HTC, Sony...)
Samsung J7 Pro
#### Cordova CLI version and cordova platform version

    cordova --version                                    # e.g. 6.0.0
    cordova platform version android                     # e.g. 4.1.1

#### Plugin version

    cordova plugin version | grep phonegap-plugin-push   # e.g. 1.5.3

**Comment by macdonst** [comment]
@gmoraiz no, it is not currently supported. You can use `notId` instead.

https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md#stacking

**Comment by gmoraiz** [comment]
Okay!!! Thank you so much!
